mitiq/pec/pec.py

```diff
@@ -266,6 +281,7 @@ def decorator(
     ) -> Callable[[QPROGRAM], Union[float, Tuple[float, Dict[str, Any]]]]:
         return mitigate_executor(
             executor,
+            None,
```

Comment:
I think `pec_decorator` and `mitigate_executor` should behave in an equivalent way. 
However `mitigate_executor` has the observable argument while `pec_decorator` doesn't.

I am in favor of adding the `observable` option to both  `pec_decorator` and `mitigate_executor`, even if it looks a bit strange to change the return type of the mitigated executor to a `float` (the result of `execute_with_pec`). 

That's the only meaningful way in which an executor that doesn't return a `float` can be "mitigated" with PEC.

If you agree, we should also change `mitiq.zne.zne_decorator()` consistently.

Reply:
Thanks for flagging. I initially did not add it because I was thinking the decorator needs to return float for this workflow. But I think the following makes sense to do:

```python
@pec_decorator(observable=obs)
def sample_bitstrings(circuit) -> MeasurementResult:
    return bitstrings
```

I will add this.

Reply:
Added.

> If you agree, we should also change mitiq.zne.zne_decorator() consistently.

In #967.